Tests/AmplitudeTests/Utilities/EventPipelineTests.swift

```diff
@@ -15,10 +15,11 @@ final class EventPipelineTests: XCTestCase {
     private var configuration: Configuration!
     private var pipeline: EventPipeline!
     private var httpClient: FakeHttpClient!
+    private var storage: PersistentStorage!
 
     override func setUp() {
         super.setUp()
-        let storage = FakeInMemoryStorage()
+        storage = PersistentStorage(storagePrefix: "event-pipeline-tests")
```

Comment:
The change merged earlier changed the interface, maybe pull the latest and update this

Reply:
thanks!  let me rebase...